cms/core/tests/test_permissions.py

```diff
@@ -0,0 +1,134 @@
+        cls.create_user(group_name=settings.PUBLISHING_ADMINS_GROUP_NAME)
+
+    def test_publishing_admin_can_access_admin(self):
+        """Check that the Publishing Admin can access the Wagtail admin."""
+        self.assertTrue(self.user.has_perm("wagtailadmin.access_admin"))
+
+    def test_publishing_admin_log_entry(self):
+        """Check that the Publishing Admin can see the logs."""
+        self.assertTrue(self.user.has_perm("wagtailcore.view_logentry"))
+
+    def test_publishing_admin_can_manage_pages(self):
+        """Check that the Publishing Admin can manage the root page and its children."""
+        for permission_type in WAGTAIL_PAGE_PERMISSION_TYPES:
+            self.page_permission_check_helper(permission_type)
+
+    def test_publishing_admin_can_manage_images(self):
+        """Check that the Publishing Admin can manage images."""
+        for permission_type in WAGTAIL_PERMISSION_TYPES:
+            self.image_collection_permission_check_helper(permission_type)
+
+    def test_publishing_admin_can_choose_images(self):
+        """Check that the Publishing Admin can choose images on pages."""
+        self.image_collection_permission_check_helper("choose")
+
+    def test_publishing_admin_can_manage_glossary_term(self):
+        self.all_permissions_check_helper("core", "glossaryterm")
+
+    def test_publishing_admin_can_manage_contact_details(self):
+        self.all_permissions_check_helper("core", "contactdetails")
+
+    def test_publishing_admin_can_manage_main_menu(self):
+        self.all_permissions_check_helper("navigation", "mainmenu")
+
+    def test_publishing_admin_can_manage_footer_menu(self):
+        self.all_permissions_check_helper("navigation", "footermenu")
+
+    def test_publishing_admin_can_manage_bundles(self):
+        """Check that the Publishing Admin can manage bundles."""
+        self.all_permissions_check_helper("bundles", "bundle")
+
+
+class PublishingOfficerPermissionsTestCase(PermissionsTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.create_user(group_name=settings.PUBLISHING_OFFICERS_GROUP_NAME)
+
+    def test_publishing_officer_can_access_admin(self):
+        """Check that the Publishing Admin can access the Wagtail admin."""
+        self.assertTrue(self.user.has_perm("wagtailadmin.access_admin"))
+
+    def test_publishing_officer_can_manage_bundle(self):
+        """Check that the Publishing Officer can manage bundles."""
+        self.all_permissions_check_helper("bundles", "bundle")
+
+    def test_publishing_officer_can_manage_pages(self):
+        """Check that the Publishing Officer can create and edit the root page and its children."""
+        for permission_type in ("add", "change"):
+            self.page_permission_check_helper(permission_type)
+
+    def test_publishing_officer_can_choose_images(self):
+        """Check that the Publishing Officer can choose images on pages."""
+        self.image_collection_permission_check_helper("choose")
+
```

Comment:
You're currently testing to check image collection permissions, but not checking for document collection permissions.

- `Publishing Admin` gets to add, change, delete + choose on both images and documents.
- `Publishing Officer` gets to choose both images and documents.

Suggestion here is to:
- Tests for `choose_document` for both `Publishing Admin` and `Publishing Officer`.
- Tests for `add_document`, `change_document`, `delete_document` for `Publishing Admin`.

Reply:
Yup, tests on the way, just finishing the UI ones now 🫡

Reply:
@kacperpONS Has this been addressed yet?

Reply:
Yup, first after this comment: [f49d5fc](https://github.com/ONSdigital/dis-wagtail/pull/128/commits/f49d5fcbfd4fddc8a046dd7713a774513dbdc50b)